backends/blocked/ceed-blocked-operator.c

```diff
@@ -561,6 +582,12 @@ static int CeedOperatorAssembleLinearQFunction_Blocked(CeedOperator op,
   // Setup
   ierr = CeedOperatorSetup_Blocked(op); CeedChk(ierr);
 
+  // Check for identity
+  if (impl->identityqf)
+    // LCOV_EXCL_START
+    return CeedError(ceed, 1, "Assembling identity qfunction does not make sense");
```

Comment:
Maybe "not supported"? I don't see why it "doesn't make sense"; it could be implemented with a fast path some day.

Reply:
In this case the qfunction is just a memcopy, so we could return a vector of 1s if desirable.

Updated the message.